aws-auth/src/main/java/software/amazon/awssdk/awsauth/credentials/AwsSystemSetting.java

```diff
@@ -43,9 +35,8 @@
 public enum AwsSystemSetting implements SystemSetting {
```

Comment:
Belongs in awscore

Reply:
This is used by some credential providers, eg:`SystemSettingsCredentialsProvider` is using it to locate the credentials.